src/data/repositories/SurveyFormD2Repository.ts

```diff
@@ -44,6 +44,8 @@ const SURVEY_NAME_DATAELEMENT_ID = "mEQnAQQjdO8";
 const SURVEY_HOSPITAL_CODE_DATAELEMENT_ID = "uAe6Mlw2XlE";
 const SURVEY_WARD_CODE_DATAELEMENT_ID = "q4mg5z04dzd";
 const SURVEY_PATIENT_CODE_DATAELEMENT_ID = "yScrOW1eTvm";
+const PATIENT_ID = "eKUbBRlR684";
+const PATIENT_CODE = "yScrOW1eTvm";
```

Comment:
reuse existing SURVEY_PATIENT_CODE_DATAELEMENT_ID for PATIENT_CODE (not a blocker, can address in next PR)